include/uapi/sound/sof/tokens.h

```diff
@@ -126,4 +126,8 @@
 #define SOF_TKN_MUTE_LED_USE			1300
 #define SOF_TKN_MUTE_LED_DIRECTION		1301
 
+/* HDA */
```

Comment:
Typo in summary line: rate amd channels -> rate and channels

Reply:
I wonder whether it makes sense to keep 2nd and 3rd patches separate. The first one doesn't really make much sense on its own. And I don't think it's worth having the same explanation again in this commit, on why this is done. 

If you keep these separate, maybe something more useful to commit description than 
"FW needs to know rate and channels." 
Maybe just add "Added support for rate and channel tokens for HDA DAI. Added in topology interface 3.16."